### PR TITLE
Navbar links solved

### LINKED ISSUE #211: [bug] Incorrect Links in Navbar
### Describe the bug

image

the links in navbar is not doing what they meant for 
roadmap is taking us to mentorship
join is taking us to roadmap

### To Reproduce

go to the website click on roadmap or join

### Expected Behavior

they should perform what they are for

### Screenshot/ Video

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Rishav1707**
## Fixes Issue

This PR fixes the following issues:

closes #211 

## Changes proposed
* Added `id = "join"` in index.jsx file.
* Added links to their respective section.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 

https://user-images.githubusercontent.com/97666287/209069866-1162e499-1265-460e-b2b2-0449ccf61cec.mp4

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
the external links should open in a new tab, add the target attribute

**Comment by Rishav1707** [comment]
But how, because target attribute is only apply in anchor link @Aadarsh805

**Review by ManishBisht777 — COMMENTED** [review]
looks good to me

**Comment by Aadarsh805** [comment]
> the external links should open in a new tab, add the target attribute

image
it's currently being used like this, so can be added

**Comment by Rishav1707** [comment]
> > the external links should open in a new tab, add the target attribute
> 
> image it's currently being used like this, so can be added

Let me try again

**Comment by Rishav1707** [comment]
> > the external links should open in a new tab, add the target attribute
> 
> image it's currently being used like this, so can be added

It is not working even after the target attribute.

target

**Comment by Aadarsh805** [comment]
> > > the external links should open in a new tab, add the target attribute
> > 
> > 
> > image it's currently being used like this, so can be added
> 
> It is not working even after the target attribute.
> 
> target

try with react <Link to='' target="_blank" />, instead of anchor tag

**Comment by Rishav1707** [comment]
> > > > the external links should open in a new tab, add the target attribute
> > > 
> > > 
> > > image it's currently being used like this, so can be added
> > 
> > 
> > It is not working even after the target attribute.
> > target
> 
> try with react , instead of anchor tag

I have tried in react also but not working

**Comment by SuperAayush** [comment]
Hey @Rishav1707!!

This has already been fixed
Thank You for the effort and contribution.

Closing the PR.

**Comment by Aadarsh805** [comment]
@Rishav1707 get issues assigned first next time

**Comment by Rishav1707** [comment]
> @Rishav1707 get issues assigned first next time

Ok @Aadarsh805
